packages/shared-influx-pieces/src/listeners/commandUsageAnalytics.ts

```diff
@@ -0,0 +1,18 @@
+import { Point } from '@skyra/influx-utilities';
+import { Piece, type ClientEvents, type ClientEventCommandContext } from '@skyra/http-framework';
+import { AnalyticsListener } from 'lib/AnalyticsListener.js';
+import { Points, Tags, Actions } from 'lib/AnalyticsSchema.js';
+
+export class UserAnalyticsEvent extends AnalyticsListener {
+	public constructor(context: Piece.Context, options: AnalyticsListener.Options) {
+		super(context, { ...options, event: 'commandSuccess' satisfies keyof ClientEvents });
```

Comment:
Does http-framework not require `event` to be `keyof ClientEvents`? If it does, `as const` will be enough.